mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -1,4 +1,26 @@
 CLASS net/minecraft/class_2246 net/minecraft/block/Blocks
+	METHOD method_26106 getLeavesSettings ()Lnet/minecraft/class_2397;
+	METHOD method_26107 getLightLevelFromBlockState (I)Ljava/util/function/ToIntFunction;
+		ARG 0 level
+	METHOD method_26109 getBedSettings (Lnet/minecraft/class_1767;)Lnet/minecraft/class_2244;
+		ARG 0 color
+	METHOD method_26110 getShulkerBoxSettings (Lnet/minecraft/class_1767;Lnet/minecraft/class_4970$class_2251;)Lnet/minecraft/class_2480;
+		ARG 0 color
+		ARG 1 settings
+	METHOD method_26115 getStemSettings (Lnet/minecraft/class_3620;)Lnet/minecraft/class_2248;
+		ARG 0 materialColor
+	METHOD method_26117 getLogSettings (Lnet/minecraft/class_3620;Lnet/minecraft/class_3620;)Lnet/minecraft/class_2465;
```

Comment:
Also `createLog(Block)`, same reasoning as piston, returns a ready-to-go `PillarBlock`